lib/tizen_sdk.dart

```diff
@@ -114,61 +113,87 @@ class TizenSdk {
   String get defaultGccVersion => '9.2';
 
   Rootstrap getFlutterRootstrap({
-    String profile = 'common-4.0',
+    @required String profile,
+    @required String apiVersion,
     @required String arch,
   }) {
-    // Defaults to wearable if the profile name is common.
-    profile = profile.replaceFirst('common', 'wearable');
+    assert(profile != null);
+    assert(apiVersion != null);
 
-    String id;
-    if (arch == 'arm64') {
-      // The arm64 build is only supported by the iot-headed-6.0 profile.
-      profile = 'iot-headed-6.0';
-      id = '$profile-device64.core';
-    } else {
-      id = '$profile-${arch == 'x86' ? 'emulator' : 'device'}.core';
+    if (profile == 'common') {
+      // The headless profile is not supported.
+      profile = 'iot-headed';
+    }
+    if (profile == 'tv') {
+      // Use the wearable rootstrap because the TV native rootstrap is not
+      // publicly available.
+      profile = 'wearable';
```

Comment:
How about providing any information to user when the requested profile or arch is changed?

Reply:
Done!